### DISCUSSION TITLE
Non-tech old man with question

### DISCUSSION THREAD

**Discussion by HuckRiley** [Stellarium/stellarium#2091] [Q&A]
I have a to-scale, white-dot on black-background chart that I would like to run against an actual star chart to see if it matches anywhere in the heavens.  If I purchase this software, will it run such a search for me?  Thanks.

**Comment by alex-w** [comment]
Stellarium is free software and it have zero price (desktop edition at least). Any planetarium may be use to compare paper chart and image on the monitor by own eyes. Automatization of search the stars patterns possible, but throug other software, such astrometry.net

**Reply by HuckRiley** [reply]
Thank you.  I'll need automatization so I will follow up on astrometry.net.  Thanks again.

**Reply by offermannu** [reply]
Hi HuckRiles

I made something myself that plate-solves an image with astrometry-net and, if successful, aligns Stellarium to the sky section using the remote API. 
You should first try to online plate-solve your images at the astrometry-net web page. You can upload your pictures [here](http://nova.astrometry.net/upload) and get them solved for free. This is really useful because you will quickly know if your images can be solved at all by the astrometry-net software and what field of view your images have. This is essential for doing the plate-sloving locally on your PC because you have to download the right index files and give the solver an idea of the FOV.

This is all not a big problem, but you have to write some scripts to control the whole thing, but I could help you there.
In my case, I mount my smartphone on the telescope and take a photo. This is transferred to my laptop and the astrometry-net software runs there. If the analysis is successful, Stellarium displays the section, so that I know where the telescope is pointing to. As I said, I already have the scripts and the whole thing is already working quite well.

Please let me know if I can help you further.

Udo

**Comment by HuckRiley** [comment]
Thanks, Udo.  I did upload three images to astronomy/net but I got a fail
on all three.  I don't fault their end.  My photographs were not of the
heavens.  Turns out they were created based upon satellite views of
prairie dog towns, edited to white dots on black backgrounds.  They looked
like the stars, but they were not.  Thank you for your insight.  Jim.

**Comment by offermannu** [comment]
You're welcome :-)
